src/lib/loadPageMetadataMap.ts

```diff
@@ -98,9 +136,14 @@ const constructBreadcrumbs = (
 };
 
 export const loadPageMetadataMap = async ({
-  includeBreadcrumbs = true,
+  includeBreadcrumbs = false,
+  includeRedirects = false,
```

Comment:
Changed these to default to `false` per your suggestion @hinzed1127. Even though the main use-case includes these, we have more use-cases where we don't.